apps/condo/domains/notification/adapters/apple/AppleMessaging.js

```diff
@@ -161,7 +161,12 @@ class AppleMessaging {
                 payload.aps.alert = { ...notification }
             }
 
-            const response = await this.sendPush(token, payload, options)
+            let response
+            for (let retryCounter = 0; retryCounter < 5; retryCounter++) {
+                response = await this.sendPush(token, payload, options)
+                if (response instanceof Error) continue
```

Comment:
Maybe add some logs?
```suggestion
                if (response instanceof Error) {
                    logger.warn({ msg: '...' })
                    continue
                }
```